### PR TITLE
Add some code to allow dns resolution

### PR DESCRIPTION
**Pull request by lofidellity**
Allow for dns resolution across vpc peers spanning multiple caccounts

### CODE REVIEW

**Comment by lofidellity** [comment]
Looks like that CI fails even on master. I don't think the linter likes having variables named outside the template using them. It's dying on `${var.stage}` which I didn't change.

File: requester.tf [hunk 1]
```diff
@@ -94,6 +99,19 @@ resource "aws_vpc_peering_connection" "requester" {
...
   tags = "${module.requester.tags}"
+
+}
+
+resource "aws_vpc_peering_connection_options" "requester" {
+  provider = "aws.requester"
+
+  # As options can't be set until the connection has been accepted
+  # create an explicit dependency on the accepter.
+  vpc_peering_connection_id = "${join("", aws_vpc_peering_connection.requester.*.id)}"
```
Comment (aknysh) [inline requester.tf]:
```suggestion
  vpc_peering_connection_id = "${join("", aws_vpc_peering_connection.accepter.*.id)}"
```

Comment (lofidellity) [reply requester.tf]: This doesn't seem to work:

Error: resource 'aws_vpc_peering_connection_options.requester' config: unknown resource 'aws_vpc_peering_connection.accepter' referenced in variable aws_vpc_peering_connection.accepter.*.id

**Comment by lofidellity** [comment]
Made the requested changes.

**Comment by aknysh** [comment]
@lofidellity  thanks.
Was it tested on AWS (`terraform apply`)?

**Comment by aknysh** [comment]
please also run `terraform fmt` on all TF files
TravisCI is failing https://travis-ci.org/cloudposse/terraform-aws-vpc-peering-multi-account/builds/490675953

**Comment by aknysh** [comment]
@lofidellity  can you please address the latest comments so we could merge it?

**Comment by lofidellity** [comment]
yes I am using this code across multiple accounts in my terraform codebase.

**Pull request merged**